### DISCUSSION TITLE
Base8 encoding binding

### DISCUSSION THREAD

**Discussion by kuettler** [janet-lang/janet#1317] [Show and tell]
Hi, here is a little janet module that adds functions for base58 encoding. As used to display bitcoin hashes. https://github.com/kuettler/janet-base58 Are these kinds of things interesting?

**Comment by sogaiu** [comment]
Somewhat related might be spork's [base64.janet](https://github.com/janet-lang/spork/blob/641b27238e073c5f5f963ec16c79b51643d3e66f/spork/base64.janet) -- that's a pure Janet implementation in contrast to `janet-base58`, IIUC.

---

Meta note: perhaps the intention of the title for this discussion item is "Base58" and not "Base8"?
